### PR TITLE
lol

### PR DESCRIPTION
**Pull request by lime360**

### CODE REVIEW

**Comment by Burgerballs** [comment]
Doesnt psych engine already support hscript tho

**Review by doggogit — COMMENTED** [review]
Psych already has HScript support, while it only being baked into Lua scripts, people have found workarounds that make it so it reads full on .hx files.

There's an already pre-existing Kade fork added here.

Only thing I approve of is the Awesome manifesto badge.

**Comment by doggogit** [comment]
> Psych already has HScript support.
> 
> There's an already pre-existing Kade fork added here.
> 
> Only thing I approve of is the Awesome manifesto badge.

Upon further review, OS Engine seems to be archived in which case means that the engine falls under the **depricated** category, in turn: it won't receive updates/won't be maintained.

**Comment by Burgerballs** [comment]
There is nothing strikingly different about OS Engine compared to Psych Engine either

**Comment by doggogit** [comment]
> There is nothing strikingly different about OS Engine compared to Psych Engine either, other than a hud that is easy to recreate

Exactly. Psych but instead of making the different UI and features in Lua, we made them in source instead.

**Comment by Burgerballs** [comment]
The awesome manifesto does say that you need to make sure that the stuff you are adding in has enough differences and features to be notable

**Comment by doggogit** [comment]
> The awesome manifesto does say that you need to make sure that the stuff you are adding in has enough differences and features to be notable

True. Apart from the badge at the top of the README, I'd say this PR would have to be closed.

**Comment by lime360** [comment]
i also made a pr to official awesome list compilation to add this list

**Comment by lime360** [comment]
> OS Engine seems to be archived in which case means that the engine falls under the depricated category
im doing a fork of it

**Comment by doggogit** [comment]
Let me reiterate - I don't think that OS Engine and Dave Engine should be in the list ***period***.

1. OS Engine, in itself, is a *fork* of Psych, we don't need you to make a *fork* of a **fork** of ***Psych***. There are good forks out there and I am sorry to say, but OS Engine, is not one of them. The only good ones that I've seen is [Ralt's Psych Fork](https://github.com/Raltyro/FNF-PsikeEngine), [The Pluma Team Psych Fork](https://github.com/Pluma-Team/FNF-PsychEngine-Pluma), [Skullbite's Psych Fork that is adapted towards one-song-mods only](https://github.com/Skullbite/OneSongPsych) and probably a few others.

Here, I will breakdown each feature of OS Engine to those who don't know what it does:
- Note skins
A pretty useful feature, however, there are a ton of Lua scripts that have this option that don't require you to download an entire engine for it.
- Showcase Mode
Literally pointless, you can hide the HUD using Psych as it *literally* already has an option for it. You can remove the BOTPLAY text with only ***one line of code***.
- Hide Score Text
Also useless, you can remove the score text with one line of code, [even the engine's owner doesn't know why you'd need this](https://user-images.githubusercontent.com/115869532/215359351-5fa3793f-4d5a-4fe0-88fd-d20289b9b954.png).
- Perfect!! Judgement
Could be useful, but still possible using scripts, no need for downloading an entire engine.
- Lane Underlay
Another useful one, however, still possible with scripts, pointless downloading an entire engine for it.
- Custom Settings in Chart Editor
Don't really know what they mean about this one.
- Literally Useless Exit Game State
It's in the name. It's fucking useless.
- Bit Changed Main Menu State
Possible with minimal source knowledge, plus a lot of people like the menus the way they are.
- Winning Icons
Cringe. (but possible with scripts as well)
- Shaders
Eric contributed to the original Psych repo, adding RuntimeShaders that were available in either late 0.5 or late 0.6 versions.
- Stripped Health Bar
They literally took the health bar from the VS. Cassette Girl mod, stripped it using Photoshop and slapped it on their own health bar.

In conclusion, OS Engine, is literally just scripts from #script-downloads from the Psych Engine Discord server thrown into source and huzzah, a new engine is born!

2. In terms of Dave Engine, only one mod uses it, and we already have a more up-to-date and general Kade Engine fork. I don't think it has any place here, as this is a list with *general* modding resources and not ***mod specific***.

My point still stands, the badge for the manifesto is the only thing alright with this Pull request.

**Comment by Burgerballs** [comment]
> Let me reiterate - I don't think that OS Engine and Dave Engine should be in the list _**period**_.
> 
> 1. OS Engine, in itself, is a _fork_ of Psych, we don't need you to make a _fork_ of a **fork** of _**Psych**_. There are good forks out there and I am sorry to say, but OS Engine, is not one of them. The only good ones that I've seen is [Ralt's Psych Fork](https://github.com/Raltyro/FNF-PsikeEngine), [The Pluma Team Psych Fork](https://github.com/Pluma-Team/FNF-PsychEngine-Pluma), [Skullbite's Psych Fork that is adapted towards one-song-mods only](https://github.com/Skullbite/OneSongPsych) and probably a few others.
> 
> Here, I will breakdown each feature of OS Engine to those who don't know what it does:
> 
> * Note skins
>   A pretty useful feature, however, there are a ton of Lua scripts that have this option that don't require you to download an entire engine for it.
> * Showcase Mode
>   Literally pointless, you can hide the HUD using Psych as it _literally_ already has an option for it. You can remove the BOTPLAY text with only _**one line of code**_.
> * Hide Score Text
>   Also useless, you can remove the score text with one line of code, [even the engine's owner doesn't know why you'd need this](https://user-images.githubusercontent.com/115869532/215359351-5fa3793f-4d5a-4fe0-88fd-d20289b9b954.png).
> * Perfect!! Judgement
>   Could be useful, but still possible using scripts, no need for downloading an entire engine.
> * Lane Underlay
>   Another useful one, however, still possible with scripts, pointless downloading an entire engine for it.
> * Custom Settings in Chart Editor
>   Don't really know what they mean about this one.
> * Literally Useless Exit Game State
>   It's in the name. It's fucking useless.
> * Bit Changed Main Menu State
>   Possible with minimal source knowledge, plus a lot of people like the menus the way they are.
> * Winning Icons
>   Cringe. (but possible with scripts as well)
> * Shaders
>   Eric contributed to the original Psych repo, adding RuntimeShaders that were available in either late 0.5 or late 0.6 versions.
> * Stripped Health Bar
>   They literally took the health bar from the VS. Cassette Girl mod, stripped it using Photoshop and slapped it on their own health bar.
> 
> In conclusion, OS Engine, is literally just scripts from #script-downloads from the Psych Engine Discord server thrown into source and huzzah, a new engine is born!
> 
> 2. In terms of Dave Engine, only one mod uses it, and we already have a more up-to-date and general Kade Engine fork. I don't think it has any place here, as this is a list with _general_ modding resources and not _**mod specific**_.
> 
> My point still stands, the badge for the manifesto is the only thing alright with this Pull request.

The OS Engine shaders are also just the broken functions from when bbpanzu made them, also OS Engine hasn't updated since like 0.5.1.

Dave Engine is really unstable, too.

**Comment by Burgerballs** [comment]
Psych engine forkers are wild man...

**Comment by doggogit** [comment]
I do apologize for coming off harsh, maybe this wasn't the right approach, however, thank you for removing the engines, I can safely approve this Pull request now.

**Review by doggogit — APPROVED** [review]

**Comment by BombasticTom** [comment]
> ok removed 2 tools from the list are you now fucking happy about this??

relax bro

**Comment by lime360** [comment]
> > ok removed 2 tools from the list are you now fucking happy about this??
> 
> relax bro

ok

**Comment by lime360** [comment]
guys this list is not going to be added to awesome because [it's not passes linting](https://github.com/sindresorhus/awesome/actions/runs/4037368459/jobs/6940618592)

**Comment by lime360** [comment]
im closing this pr since @EliteMasterEric included awesome badge thank you so much!!!

**Comment by EliteMasterEric** [comment]
The Awesome link was wrong anyway (it needs to be a badge and other links are redundant).
